test/test_sources.py

```diff
@@ -91,5 +115,5 @@ def test_multisolution_calls_search(self, mock_view):
     observations = self.make_random_observations(experiment, 5)
     source = SPESource(experiment)
     point, task_cost = source.next_point(observations)
-    assert point == fake_point
+    assert point == [fake_point]
     assert task_cost is None
```

Comment:
can you also add the following test to `test_sources.py`

```
class TestAllSources(UnitTestsBase):
  @pytest.mark.parametrize("source_class", [RandomSearchSource, GPSource, SPESource])
  def test_exhausted_options(self, source_class):
    num_observations = 12
    experiment_meta = self.get_experiment_feature("categorical")
    experiment = LocalExperimentBuilder(experiment_meta)
    source = source_class(experiment)
    observations = self.make_random_observations(experiment, num_observations)
    suggestion = source.get_suggestion(observations)
    self.assert_valid_suggestion(suggestion, experiment)
```

you will need add the following to `test/constants.py`
```
EXPERIMENT_META_CATEGORICAL = dict(
  parameters=[PARAMETER_CATEGORICAL],
  metrics=DEFAULT_METRICS,
  observation_budget=10,
)
```
```
 ALL_META = dict(
  categorical=EXPERIMENT_META_CATEGORICAL,
```